_developer-content/import-api/guides/sequencing-data-for-the-import-api.md

```diff
@@ -0,0 +1,204 @@
+---
+# -------------------------- #
+#          PAGE INFO         #
+# -------------------------- #
+
+title: Sequencing Data for the Stitch Import API
+permalink: /developers/import-api/guides/sequencing-data-for-the-import-api
+summary: &summary "Learn how to sequence data for the Import API."
+
+doc-type: "concept"
+
+product-type: "import-api"
+content-type: "guide"
+content-id: "sequence-data"
+
+key: "import-api-sequence-data"
+
+layout: general
+sidebar: on-page
+
+icon: replication
+order: 4
+
+
+# -------------------------- #
+#      GUIDE PAGE INFO       #
+# -------------------------- #
+
+## This is used only on the /import-api/guides page.
+
+display-title: "Sequencing data for the Import API"
+description: *summary
+
+
+# -------------------------- #
+#   RELATED SIDEBAR LINKS    #
+# -------------------------- #
+
+related:
+  - title: "Import API reference"
+    link: "{{ link.import-api.api | prepend: site.baseurl }}"
+
+  - title: "Structuring data for the Import API"
+    link: "{{ link.import-api.guides.structure-data | prepend: site.baseurl }}"
+
+  - title: "Understanding loading behavior"
+    link: "{{ link.destinations.storage.loading-behavior | prepend: site.baseurl }}"
+
+
+# -------------------------- #
+#            INTRO           #
+# -------------------------- #
+
+intro: |
+  {% include misc/data-files.html %}
+
+  Stitch uses record sequencing to determine in what order extracted records are considered for loading. This practice, along with de-duping data using Primary Keys, ensures that only the most recent version of a record is loaded into a destination. Newer data points can replace older ones, but not vice versa.
+
+  Stitch uses the `sequence` property on records pushed to the Import API to accomplish this. Every record in a request must have this property.
+
+  In this guide, we'll cover:
+
+  {% for section in page.sections %}
+  - [{{ section.summary }}](#{{ section.anchor }})
+  {% endfor %}
+
+
+# -------------------------- #
+#           CONTENT          #
+# -------------------------- #
+
+sections:
+  - title: "Understanding record sequencing"
+    anchor: "understand-record-sequencing"
+    summary: "The basics of record sequencing"
+    content: |
+      {% for subsection in section.subsections %}
+      - [{{ subsection.title }}](#{{ subsection.anchor }})
+      {% endfor %}
+
+    subsections:
+      - title: "Sequencing and Replication Keys"
+        anchor: "sequence-replication-keys"
+        content: |
+          Think of `sequence` like a [Replication Key]({{ link.replication.rep-keys | prepend: site.baseurl }}).
+
+          For the Import API, it functions similarly to an `updated_at` property: Only records that have a `sequence` value greater than **[TODO- Confirm that this has to be greater than the saved sequence value for the TABLE and not the record. Pretty sure this is the case, but it should be confirmed.]**
```

Comment:
It has to be greater than the saved sequence number for any existing record with the same primary key. You can see the logic for removing records from the batch that _don't_ have a greater sequence number than the existing record here: https://github.com/stitchdata/loader-snow/blob/master/src/clj/com/stitchdata/loader_snow/models/ddl.clj#L298